tools/src/main/scala/caliban/tools/stitching/RemoteResolver.scala

```diff
@@ -0,0 +1,23 @@
+package caliban.tools.stitching
+
+import zio._
+
+case class RemoteResolver[-R, +E, -A, +B](
```

Comment:
I'm not sure if this is stack safe or not.

I suspect it might not be, but don't think my Scala knowledge is enough to really know how I would make it stack safe either. But maybe it won't matter because these chains shouldn't be too deep.